mimicpy/core/_qmhelper.py

```diff
@@ -32,6 +35,40 @@ def index(qmids, name, space_len=6, col_len=15):
     
     return index
 
+def check_mdp(mdp):
+    nsteps = 1000
+    dt = 0.0001
+    
+    if mdp is None:
+        return nsteps, dt/hartree_to_ps, ""
```

Comment:
Is this for some later call of the function? Maybe throw a MdpNotFound Error and catch it?

Reply:
See the answer for the previous question